docs/topics/Gateway.md

```diff
@@ -177,6 +177,115 @@ If successful, the gateway will respond by replaying all missed events in order,
+
+### List of Intents
+
+```
+GUILDS (1 << 0)
+  - GUILD_CREATE
+  - GUILD_DELETE
+  - GUILD_ROLE_CREATE
+  - GUILD_ROLE_UPDATE
+  - GUILD_ROLE_DELETE
+  - CHANNEL_CREATE
+  - CHANNEL_UPDATE
+  - CHANNEL_DELETE
+  - CHANNEL_PINS_UPDATE
+
+GUILD_MEMBERS (1 << 1)
+  - GUILD_MEMBER_ADD
+  - GUILD_MEMBER_UPDATE
+  - GUILD_MEMBER_REMOVE
+
+GUILD_BANS (1 << 2)
+  - GUILD_BAN_ADD
+  - GUILD_BAN_REMOVE
+
+GUILD_EMOJIS (1 << 3)
+  - GUILD_EMOJIS_UPDATE
+
+GUILD_INTEGRATIONS (1 << 4)
+  - GUILD_INTEGRATIONS_UPDATE
+
+GUILD_WEBHOOKS (1 << 5)
+  - WEBHOOKS_UPDATE
+
+GUILD_INVITES (1 << 6)
+  - INVITE_CREATE
+  - INVITE_DELETE
+
+GUILD_VOICE_STATES (1 << 7)
+  - VOICE_STATE_UPDATE
+
+GUILD_PRESENCES (1 << 8)
+  - PRESENCE_UPDATE
+
+GUILD_MESSAGES (1 << 9)
+  - MESSAGE_CREATE
+  - MESSAGE_UPDATE
+  - MESSAGE_DELETE
+
+GUILD_MESSAGE_REACTIONS (1 << 10)
+  - MESSAGE_REACTION_ADD
+  - MESSAGE_REACTION_REMOVE
+  - MESSAGE_REACTION_REMOVE_ALL
+  - MESSAGE_REACTION_REMOVE_EMOJI
+
+GUILD_MESSAGE_TYPING (1 << 11)
+  - TYPING_START
+
+DIRECT_MESSAGES (1 << 12)
+  - CHANNEL_CREATE
+  - MESSAGE_CREATE
+  - MESSAGE_UPDATE
+  - MESSAGE_DELETE
+  - CHANNEL_PINS_UPDATE
+
+DIRECT_MESSAGE_REACTIONS (1 << 13)
+  - MESSAGE_REACTION_ADD
+  - MESSAGE_REACTION_REMOVE
+  - MESSAGE_REACTION_REMOVE_ALL
+  - MESSAGE_REACTION_REMOVE_EMOJI
+
+DIRECT_MESSAGE_TYPING (1 << 14)
+  - TYPING_START
+```
+
+Any [events not defined in an intent](#DOCS_TOPICS_GATEWAY/commands-and-events-gateway-events) are considered "passthrough" and will always be sent to you. If you specify an `intent` value in your `IDENTIFY` payload that is invalid, the socket will close with a [`4012` close code](#DOCS_TOPICS_OPCODES_AND_STATUS_CODES/gateway-gateway-close-event-codes). An invalid intent value can be one of:
+
+- A miscalculated bit flag
+- Specifying a privileged intent for which you have not opted in for your bot
+- Specifying a privileged intent for which you have not been whitelisted, and your bot is in more than the current limit for unwhitelisted access
+
+### Privileged Intents
+
+> danger
+> As of <date>, Gateway Intents are optional in v6 **except for `GUILD_PRESENCES`**. If you wish to get `PRESENCE_UPDATE` events, you **must** declare the intent. Otherwise, you will no longer receive `PRESENCE_UPDATE` events on the v6 gateway after <date>
+
+> warns
+> The current limit for specifying privileged intents without whitelisting is 100 guilds.
+
+Some intents require whitelisting once your bot scales beyond a certain size. Those intents are:
+
+- `GUILD_PRESENCES`
+
+The current limit for specifying these privileged intents without being whitelisted is 100 guilds. That means you may specify the above intents without restriction until you hit the current limit, at which point your bot will not be able to join any more guilds without either being whitelisted or dropping the privileged intents for which you have not been whitelisted.
```

Comment:
This is already mentioned as a warning label above. Perhaps delete the warning label or move it down to here?